pkg/controller/pxc/secrets.go

```diff
@@ -75,7 +79,8 @@ const (
 		"0123456789"
 )
 
-func generatePass() ([]byte, error) {
+//GeneratePass generate random password
+func GeneratePass() ([]byte, error) {
```

Comment:
why is this method became exported?

Reply:
fixed https://github.com/percona/percona-xtradb-cluster-operator/pull/436/commits/cd59bf0031fcfa410a6f073ef55301f1f0d3c536